RELEASES.md

```diff
@@ -668,3 +668,15 @@
 10.4.2 2019-10-23
 -----------------
   - lint prettier tests
+
+10.4.3 2019-11-11
+-----------------
+  - cache fixes - clearTimeout on persisted cache
+  - test: user deletion and recreation token reuse (test/integration/security/access_sanity)
+
+10.4.4 - skipped (on different branch)
+--------------------------------------
+
+10.4.5 2020-03-08
```

Comment:
Please note that 10.6.0 has been released. Can we please keep the master-10 track clean as it seems that is going to be alive for quite a while longer.